examples/twittermap/web/public/javascripts/map/services-multilayer-countmap.js

```diff
@@ -0,0 +1,978 @@
+angular.module("cloudberry.map")
+  .service("multilayerCountmap", function($http, $q, $compile, cloudberry, cloudberryConfig, leafletData,
+                                          moduleManager, MapResultCache, cloudberryClient, queryUtil, Cache,
+                                          TimeSeriesCache){
+    
+    function setCenterAndBoundry(features) {
+      for(var id in features){
+        if ({}.hasOwnProperty.call(features, id)) {
+          var minLog = Number.POSITIVE_INFINITY;
+          var maxLog = Number.NEGATIVE_INFINITY;
+          var minLat = Number.POSITIVE_INFINITY;
+          var maxLat = Number.NEGATIVE_INFINITY;
+          if(features[id].geometry.type === "Polygon") {
+            features[id].geometry.coordinates[0].forEach(function(pair) {
+              minLog = Math.min(minLog, pair[0]);
+              maxLog = Math.max(maxLog, pair[0]);
+              minLat = Math.min(minLat, pair[1]);
+              maxLat = Math.max(maxLat, pair[1]);
+            });
+          } else if( features[id].geometry.type === "MultiPolygon") {
+            features[id].geometry.coordinates.forEach(function(array){
+              array[0].forEach(function(pair){
+                minLog = Math.min(minLog, pair[0]);
+                maxLog = Math.max(maxLog, pair[0]);
+                minLat = Math.min(minLat, pair[1]);
+                maxLat = Math.max(maxLat, pair[1]);
+              });
+            });
+          }
+          features[id].properties["centerLog"] = (maxLog + minLog) / 2;
+          features[id].properties["centerLat"] = (maxLat + minLat) / 2;
+        }
+      }
+    }
+  
+    function loadCityJsonByBound(onEachFeature,instance){
```

Comment:
Per my understanding, now there are two `loadCityJsonByBound` functions defined and called in `services-multilayer-countmap.js` and `controllers.js` respectively. But I wonder why the same logic called twice in two places?